include/aos/common/config.hpp

```diff
@@ -155,4 +155,88 @@
 #define AOS_CONFIG_MONITORING_POLL_PERIOD_SEC 10
 #endif
 
+/**
+ * Certificate public key len.
+ */
+#ifndef AOS_CONFIG_CRYPTO_CERT_PUB_KEY_LEN
+#define AOS_CONFIG_CRYPTO_CERT_PUB_KEY_LEN 2048
+#endif
+
+/**
+ * Certificate private key len.
+ */
+#ifndef AOS_CONFIG_CRYPTO_CERT_PRIV_KEY_LEN
+#define AOS_CONFIG_CRYPTO_CERT_PRIV_KEY_LEN 2048
+#endif
+
+/**
+ * Certificate serial number len.
+ */
+#ifndef AOS_CONFIG_CRYPTO_CERT_SERIAL_LEN
+#define AOS_CONFIG_CRYPTO_CERT_SERIAL_LEN 40
+#endif
+
+/**
+ * Certificate issuer len.
+ */
+#ifndef AOS_CONFIG_CRYPTO_CERT_ISSUER_ID_LEN
+#define AOS_CONFIG_CRYPTO_CERT_ISSUER_ID_LEN 40
+#endif
+
+/**
+ * Max length of alternative DNS module name.
+ */
+#ifndef AOS_CONFIG_CRYPTO_DNS_NAME_LEN
+#define AOS_CONFIG_CRYPTO_DNS_NAME_LEN 42
+#endif
+
+/**
+ * Max number of alternative DNS names for a module.
+ */
+#ifndef AOS_CONFIG_CRYPTO_ALT_DNS_NAMES_MAX_COUNT
+#define AOS_CONFIG_CRYPTO_ALT_DNS_NAMES_MAX_COUNT 10
+#endif
+
+/**
+ * Certificate subject size.
+ */
+#ifndef AOS_CONFIG_CRYPTO_CERT_SUBJECT_SIZE
+#define AOS_CONFIG_CRYPTO_CERT_SUBJECT_SIZE 24
```

Comment:
Don't think 24 is enough. 256 for now. Also, it probably should be equal to `AOS_CONFIG_CRYPTO_CERT_ISSUER_SIZE`. Need to be clarified.